libr/core/cmd_print.c

```diff
@@ -4354,13 +4355,77 @@ static const char* bits_to_c_code_fmtstr(int bits) {
 	}
 }
 
+static void print_c_instructions(RPrint *p, ut64 addr, const ut8 *buf, int len, int ws) {
```

Comment:
if this needs to be moved in RPrint, do it now, otherwise keep it here but it needs some cleaning. `ws`, for example, can only have 1 as value, so all the operations on ws are useless.

Reply:
though I fail to see how this would be useful in a library function such as RPrint, since it implements a very particular command which I find it hard to reuse anywhere else.

Reply:
the function takes rprint as argument and it performs a very basic thing like hexdumping with a comment

ive moved the thing into r_util